scripts/apps/settings/settings.tsx

```diff
@@ -23,23 +23,15 @@ class SettingsComponent extends React.Component<IProps, IState> {
             loading: true,
             flatMenuItems: [],
         };
-
-        this.gettext = this.gettext.bind(this);
-    }
-
-    gettext(text) {
-        const {gettextCatalog} = this.props;
-
-        return gettextCatalog.getString(text);
     }
 
     componentDidMount() {
         const {$route, pageTitle, superdesk} = this.props;
         const currentRoute = $route.current;
 
-        pageTitle.setUrl(capitalize(this.gettext('Settings')));
+        pageTitle.setUrl(capitalize(gettext('Settings')));
         if (currentRoute.$$route.label !== 'Settings') {
-            pageTitle.setWorkspace(capitalize(this.gettext(currentRoute.$$route.label)));
+            pageTitle.setWorkspace(capitalize(gettext(currentRoute.$$route.label)));
```

Comment:
Do not use `gettext` for dynamic values. Instead an alias of `gettext` should be used so the code inside `gettext` isn't extracted and sent to Transifex.

It doesn't look like it makes sense to translate this at all.